scripts/front-end.sh

```diff
@@ -1 +1 @@
-sudo cp -rf src/main/webapp/WEB-INF/view /var/lib/tomcat7/webapps/protocolanalyzer/WEB-INF
\ No newline at end of file
+sudo cp -rf src/main/webapp/WEB-INF/view /opt/tomcat/webapps/protocolanalyzer/WEB-INF
```

Comment:
when I tried to load the application via tomcat8 manager GUI, it loads successfully, but loads at localhost:8080/protocolanalyzer-1.0-SNAPSHOT location. This might be due to incompatibility of context.xml between tomcat 6 and tomcat 8.

The message given out by tomcat8 manager is as follows.
```
OK - Reloaded application at context path [protocolanalyzer-1.0-SNAPSHOT]
```